app/lib/constants/colors.ts

```diff
@@ -102,6 +102,14 @@ export const colors = {
 		statusBackgroundWarning: '#FFECAD',
 		statusFontOnWarning: '#B88D00',
 		overlayColor: '#1F2329B2',
+		buttonBackgroundPrimaryDefault: '#156FF5',
```

Comment:
Can you make sure to apply the same naming from here? https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/feat.new-composer/app/lib/constants/colors.ts#L24

Not asking to apply all colors, just trying to prevent differences like `buttonFontOnPrimary`

Reply:
just the dark/black colors of buttonBackgroundSecondaryDefault are different and I used the same color as you set on new-composer.

```
- buttonBackgroundSecondaryDefault: '#353B45',
to
+ buttonBackgroundSecondaryDefault: '#2F343D',
```